lib/log.dart

```diff
@@ -30,3 +33,45 @@ bool debugLog(String message) {
   }());
   return true;
 }
+
+/// Display an error message.
```

Comment:
How about "Display an error message in a [SnackBar]." I like how the summary line on `reportErrorToUserInDialog`'s dartdoc says how the error is displayed.